lib/wine_o_inventory_api/products.ex

```diff
@@ -0,0 +1,104 @@
+defmodule WineOInventoryApi.Products do
+  @moduledoc """
+  The Products context.
+  """
+
+  import Ecto.Query, warn: false
+  alias WineOInventoryApi.Repo
+
+  alias WineOInventoryApi.Products.Wine
+
+  @doc """
+  Returns the list of wines.
+
+  ## Examples
+
+      iex> list_wines()
+      [%Wine{}, ...]
+
+  """
+  def list_wines do
```

Comment:
Seems like for this story this is really all we need?

Reply:
Yes. I ended up using phoenix's context generator, which generated the context, schema for `Wine`, and all the tests.

Reply:
So you're going to remove all of the code not related to getting a list of wines, correct?

Reply:
Yes, I'll go ahead and remove the code not related to getting a list of wines and remove tests not related to getting a list of wines as well.